DependencyInjection/DoctrineExtension.php

```diff
@@ -745,6 +745,7 @@ protected function loadOrmEntityManager(array $entityManager, ContainerBuilder $
             ->setArguments([
                 new Reference(sprintf('doctrine.dbal.%s_connection', $entityManager['connection'])),
                 new Reference(sprintf('doctrine.orm.%s_configuration', $entityManager['name'])),
+                new Reference(sprintf('doctrine.dbal.%s_connection.event_manager', $entityManager['connection'])),
```

Comment:
I'm really not sure about what I'm saying but… should we really reuse the DBAL's event manager? Or should there be a separate event manager for the ORM?

Reply:
I was also thinking about it... but currently the connection event manager is used. So it would be a change in behavior? If someone replaces/extends it or whatever?

Reply:
Using a separate event manager will require changing the way we register events (we will need separate tags for DBAL vs ORM events) and this will be a BC break. So we cannot do it in a minor version of the bundle.
Long term, it might make sense though.